plugins/tail.yml

```diff
@@ -0,0 +1,28 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: tail
+spec:
+  version: "v0.9.1"
+  platforms:
+  - uri: https://github.com/boz/kail/releases/download/v0.9.1/kail_0.9.1_darwin_amd64.tar.gz
+    sha256: 3824de171a73c773649401f2bdccbd138f0022333c69e39e4f194e8ebd08290b
+    bin: kail
+    files:
+    - from: "kail"
+      to: "."
+    selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin]}
+  - uri: https://github.com/boz/kail/releases/download/v0.9.1/kail_0.9.1_linux_amd64.tar.gz
+    sha256: 6932b86f59c4b2cc73eb7aad889d3479566dfb4034d89a2d5f426722cdc5b780
+    bin: kail
+    files:
+    - from: "kail"
+      to: "."
+    selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [linux]}
+  homepage: https://github.com/boz/kail
+  shortDescription: Kubernetes log viewer
```

Comment:
To me this sounds a bit too generic. I can show logs with `kubectl logs`, so how is this better?

Looking at your homepage, it seems that this is the unique selling point:
> Match pods by service, replicaset, deployment, and others. Adjusts to a changing cluster ...

Can you condense that further, so that this fits in one line?